### ISSUE TITLE
[FR]: Store relation in frontmatter property instead of inline data

### ISSUE THREAD

**Issue by despens** [RafaelGB/obsidian-db-folder#1002]
### Contact Details

dragan.espenschied@rhizome.org

### Present your request

In version 3.5.0, relations seems to be always created as inline data. It would be great if relations could be expressed as a regular frontmatter property.

### For which platform do you request this request??

Desktop

**Comment by stokkes** [comment]
Adding a +1 to this.

Is this by design or a bug? I (and I'm sure many others) would much prefer having everything in the frontmatter and not inline. Especially with the recent Obsidian update for Properties, it makes things a lot cleaner.

Can we get this upated(fixed?)

**Comment by FlipH19Switch** [comment]
Yes I would like to see this too. It would help with keeping pages tidy when publishing them using the [Digital Garden plugin](https://github.com/oleeskild/Obsidian-Digital-Garden).

**Comment by NibiruanChild** [comment]
+1
I'm here because of this problem

**Comment by AnthonyMansour** [comment]
I would also love this. +1

It seems like [Relations were forced to be inline ](https://github.com/RafaelGB/obsidian-db-folder/blob/f7038d1fe8f6e4237a7cd2e3f82a040826c73f07/src/stateManagement/columns/handlers/AlterColumnTypeAction.ts#L59) when they were first added cause Obsidian doesn't support parsing multiple links in a `text` frontmatter type.
It gets all garbled up like this: 
![image](https://github.com/user-attachments/assets/23c8dcfb-bcd9-48b7-8fe7-0624c94b80cd)

But Obsidian has the `List` frontmatter type that is formatted like this in YAML:
```
example_property: 
  - "[[example page 1]]"
  - "[[example page 2]]"
  - "[[etc]]"
```

So it could easily be implemented in this plugin to use that format for `Relations`.

Hopefully someone gets around to adding it, but if not I'll find some time in a few months.

**Comment by cvvvm** [comment]
Until there's a fix - you can fix relations being created as inline instead of front-matter by:
1. Going into the markdown view for the table
2. Finding related field config
3. Manually change `inline` property to "false"

Alternatively, I've found you can also create the field as `text`, and after creating change to `relation`.
